.gitmessage

```diff
@@ -0,0 +1,23 @@
+# [MODULE] A short but descriptive summary (#pr-num)
+
+# An awesome description of WHY you did this work.
```

Comment:
I feel like:
*`WHY` should usually be somewhat minimal, as detailed in JIRA Issue or GitHub PR or self-explanatory.
*`WHAT` details interest me most...

Reply:
The "what" work can also be viewed by looking at the diff/commit...
The "why" should be more explicit (maybe like a summary of some perhaps much more detailed description that can be found in some JIRA issue or GitHub PR) because one shouldn't have to open his webbrowser everytime when reading a git log.

Reply:
Also, please replace "awesome" by something like "correctly worded", "understandable"/"intelligible", etc...

Reply:
@HBelusca reworded.